atc/exec/get_step.go

```diff
@@ -205,8 +207,20 @@ func (step *GetStep) run(ctx context.Context, state RunState, delegate GetDelega
 
 	containerOwner := db.NewBuildStepContainerOwner(step.metadata.BuildID, step.planID, step.metadata.TeamID)
 
+	processCtx := ctx
+	if step.plan.Timeout != "" {
```

Comment:
A minor issue and i have mixed feeling about this but this could be extracted into a shared method.

Reply:
Yeah, now that this is consistent between all the steps it might be easy to DRY it up. I'll take a look!